modules/ROOT/pages/x12-receive-read-settings.adoc

```diff
@@ -1,148 +1,162 @@
+| Enabled
 
-|Fail document when invalid character in value
-|If selected, a transaction with invalid characters is rejected. If unselected, the characters are either passed through or substituted and the transaction is not rejected. In either case, the error is reported in the 997 acknowledgments.
-|Disabled
+|Fail when invalid character in value
+a| * If selected, the host rejects transactions with invalid characters.
+* If unselected, the characters are either passed through or substituted.
 
-|Fail document if value is repeated too many times
-|If selected, a transaction with values that are repeated too many or too few times are rejected. If unselected, the values are accepted and the transaction is not rejected. In either case, the error is reported in the 997 acknowledgments.
 |Enabled
 
-|Fail document if unknown segments are used
-|If selected, a transaction containing any unknown segments is rejected. If unselected, unknown segments are ignored and the transaction is not rejected. In either case, the error is reported in the 997 acknowledgments.
+|Fail if value is repeated too many times
+a|* If selected, the host rejects transactions with values that are repeated too many or too few times.
+*Vijay, is the "too few times" correct? It is in the current text.*
+* If unselected, the values are accepted.
 |Enabled
 
-|Fail document when segments are out of order
-|If selected, a transaction with segments that are out of order is rejected. If unselected, the segment can be reordered the transaction is not rejected. In either case, the error is reported in the 997 or 999 acknowledgments.
-|Enabled
+|Fail if unknown segments are used
+a|* If selected, the host rejects transactions that contain unknown segments.
+* If unselected, If unselected, the host ignores unused segments.
+|Disabled
 
-|Fail document when too many repeats of a segment
-|A transaction with a segment repeated too many times is rejected. The error is reported in the 997 or 999 acknowledgments.
-|Enabled
+|Fail when segments are out of order
+a|* If selected, the host rejects transactions that have out-of-order segments.
+* If unselected, the segments are reordered.
+| Enabled
 
-|Fail document when unused segments are included
-|If selected, a transaction containing segments marked as unused in the schema is rejected. If unselected, the transaction is not rejected and the unused segments are ignored. In either case, the error is reported in the 997 or 999 acknowledgments.
+|Fail when too many repeats of segment
+| If selected, the host rejects transactions with segments repeated too many times.
 |Enabled
+
+|Fail when unused segments are included
+a|* If selected, the host rejects transactions containing segments marked as unused in the schema.
+* If unselected, the host ignores unused segments.
++
+*Vijay, what is the difference between this option and the "Fail if unknown segments are used* setting? Both are in the interface. Is this a bug?*
```

Comment:
@dhirsch1 - It's not a bug.. in the EDI schema definitions, some segments may be included in the schema, but defined as "Unused". 
Unknown segments - segments that are not even defined in the schema.
Unused segments - segment is defined, but marked as Unused.